pootle/.pootle_fs/tmp/README

```diff
@@ -1 +1,3 @@
 This directory (pootle_fs) is the working directory for pootle_fs.
+
+NEVER touch anything on this directory.
```

Comment:
i think we neen some more information here

this dir has the potential to get very large, so we need to communicate that they might want/need to place it somewhere specific and make the necessary adjustments to their settings

also rem that this folders ends up in an egg dir in the middle of nowhere in a production env, so im thinking there needs to be docs elsewhere explaining all of this